### ISSUE TITLE
Market page mobile issues

### ISSUE THREAD

**Issue by Robiquet** [zeitgeistpm/ui#2281]
<img width="486" alt="image" src="https://github.com/zeitgeistpm/ui/assets/4950844/e995cd73-8c57-49f8-9d1c-32857b08730c">
<img width="489" alt="image" src="https://github.com/zeitgeistpm/ui/assets/4950844/82af01fc-dd5e-4236-949c-a68aece92c5f">

**Comment by saboonikhil** [comment]
Yeah I came across it very briefly. Wasn't able to reproduce though.
